### ISSUE TITLE
Request: select all sats

### ISSUE THREAD

**Issue by novakmi85** [rt-bishop/Look4Sat#4]
Could you please add "SELECT ALL" feature to select all satellites? Thank you. (BTW: Great software, thank you very much!)

**Comment by rt-bishop** [comment]
Hey) Actually working on it at the moment, as wel as on the search function. Will add more satellites after that.

**Comment by trebor-56** [comment]
Option to centeralise map 3 options should be ample 
as is, Africa, (how it is now) or South America or AU, set as default, or to Location (full view) 

When On active sat (Bullseye and Data) an Icon to quickly switch to map for that Sat
and on map and Icon to go back to data.
Thanks

**Comment by rt-bishop** [comment]
> Option to centeralise map 3 options should be ample
> as is, Africa, (how it is now) or South America or AU, set as default, or to Location (full view)
> 
> When On active sat (Bullseye and Data) an Icon to quickly switch to map for that Sat
> and on map and Icon to go back to data.
> Thanks

That's an interesting proposition. I'll definitely look into that. Thanks!

**Comment by rt-bishop** [comment]
I've just recently released v1.3.0 with search feature, select all/none button and additional sats. By additional sats I mean all currently active sats from celestrak, which is an INSANE amount. Don't forget to update your TLE file via the main menu. I tried to calculate passes for all of them... just be patient, it may take slightly under a minute. Not sure if it's that useful though. I look forward to receiving your feedback. Hope I didn't break anything.
